cfy_cluster_manager/utils.py

```diff
@@ -77,7 +77,8 @@ def copy(source, destination):
 
 def move(source, destination):
     ensure_destination_dir_exists(destination)
-    sudo(['mv', source, destination])
+    sudo(['cp', source, destination])
+    sudo(['rm', source])
```

Comment:
Why did you stop using mv here?
Also worth noting that this now won't work correctly with directories.

Reply:
Right! That's a great comment as it should work with directories. Will change it bask.